projects/components/src/skeleton/skeleton.component.ts

```diff
@@ -0,0 +1,65 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+interface ContainerClass {
+  skeleton: boolean;
+  'skeleton-rectangle': boolean;
```

Comment:
use camelCasing for the property keys. This will also help you avoid string usages in L56-62.

Reply:
These property names are used as class names, does that still apply ? camel case class names ?